moveit2_tutorials.repos

```diff
@@ -1,9 +1,5 @@
 repositories:
-  moveit2_tutorials:
```

Comment:
Do any of the instructions need to be updated by removing this? I think the readme expects the repo itself to be included in this file. I will try to dig around a little and see.

Reply:
Good catch, fixing it.